types/src/on_chain_config/aptos_features.rs

```diff
@@ -32,6 +32,7 @@ pub enum FeatureFlag {
     GAS_PAYER_ENABLED = 22,
     APTOS_UNIQUE_IDENTIFIERS = 23,
     BULLETPROOFS_NATIVES = 24,
+    APTOS_STDLIB_PATCH = 25,
```

Comment:
Can we name this something more specific like SIGNER_PRINT_FORMAT_FIX (but better)?

Reply:
Yes, I am done changing it to `SIGNER_NATIVE_FORMAT_FIX`.